File: Searching/LinearSearch.js
function linearSearch(arr, item) {
    for (let i = 0; i < arr.length; i++) {
        if (arr[i] === item) {
            console.log(item + " Found at index : " + i);
            return;
        }
    }
    console.log("Not found");
}

let arr = [12, 5, 18, 25, -3, 19];
let item = 25;
linearSearch(arr, item)

File: Searching/LinearSearch.c
#include <stdio.h>

void LinearSearch(int arr[], int len, int item)
{
    for(int i=0;i < len;i++)
    {
        if(arr[i] == item)
        {
        printf("%d Found at index %d", item, i);
        return;
        }
    }
    printf("Not Found");
}

int main() 
{
    int arr[] = {10, 20, 30, 40, 50};
    int len = sizeof(arr)/sizeof(arr[0]);
    int item = 40;
    LinearSearch(arr, len, item);
    return 0;
}